### PR TITLE
#1 Initial commit

### PR DESCRIPTION
**Pull request by JPerzanowski**

### CODE REVIEW

File: src/com/javafee/tasks/task1/Main.java [hunk 1]
```diff
@@ -0,0 +1,208 @@
+package com.javafee.tasks.task1;
+
```
Comment (JaRutkowski) [inline src/com/javafee/tasks/task1/Main.java:2]: Are the tasks already finished?

**Review by JaRutkowski — CHANGES_REQUESTED** [review]

File: src/com/javafee/tasks/task1/Main.java (hunk 1 above)
Comment (JPerzanowski) [reply src/com/javafee/tasks/task1/Main.java:2]: Yes

Comment (JaRutkowski) [reply src/com/javafee/tasks/task1/Main.java:2]: Please remove the task 1 related classes and move them to the task branch.
Other staff could be committed.

Comment (JaRutkowski) [reply src/com/javafee/tasks/task1/Main.java:2]: Ok, let's commit it together, and then create the branch.

**Review by JaRutkowski — APPROVED** [review]

**Pull request merged**